### PR TITLE
[zh] Update kep link and release version for dockershim removal in faq

### PR DESCRIPTION
**Pull request by mengjiao-liu**
Keep the content consistent with the English version.

ref issue #29776
en PR: https://github.com/kubernetes/website/pull/29780

Updates the dockershim FAQ blog:

fixes the link to the kep
added an update that dockershim is scheduled to be removed in 1.24 according to the KEP https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2221-remove-dockershim. The release of kubelet without dockershim is targeted for 1.24

### CODE REVIEW

File: content/zh/blog/_posts/2020-12-02-dockershim-faq.md [hunk 1]
```diff
@@ -78,12 +79,16 @@ startup if using Docker as the runtime.
...
+dockershim would be 1.23 in late 2021.
+_Update_: removal of dockershim is scheduled for Kubernetes v1.24, see 
+[Dockershim Removal Kubernetes Enhancement Proposal][drkep].
+We will be working closely with vendors and other ecosystem groups to ensure a smooth transition and will evaluate 
+things as the situation evolves.
 -->
 考虑到此改变带来的影响，我们使用了一个加长的废弃时间表。
 在 Kubernetes 1.22 版之前，它不会被彻底移除；换句话说，dockershim 被移除的最早版本会是 2021 年底发布的 1.23 版。
+_更新_：dockershim 计划在 Kubernetes 1.24 版被移除，
+请参阅 [移除 Kubernetes 增强方案 Dockershim][drkep]。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-12-02-dockershim-faq.md:91]:
```suggestion
请参阅[移除 Kubernetes 增强方案 Dockershim][drkep]。
```

Comment (mengjiao-liu) [reply content/zh/blog/_posts/2020-12-02-dockershim-faq.md]: Thanks for catch! Updated.

**Pull request merged**